colin/core/ruleset/loader.py

```diff
@@ -27,10 +27,10 @@
 
 def get_ruleset_struct_from_fileobj(fileobj):
     try:
-        logger.debug("Loading ruleset from file '{}'.".format(fileobj.name))
+        logger.debug(f"Loading ruleset from file '{fileobj.name}'.")
         return RulesetStruct(yaml.safe_load(fileobj))
     except Exception as ex:
-        msg = "Ruleset file '{}' cannot be loaded: {}".format(fileobj.name, ex)
+        msg = f"Ruleset file '{fileobj.name}' cannot be loaded: {ex}"
         logger.error(msg)
```

Comment:
Use % formatting in logging functions and pass the % parameters as arguments instead of f-string

Reply:
nevermind; msg is used also in Exception